liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -712,9 +714,11 @@ private void getServletNames(Archive archive, List<String> servletNames) throws
                      ba = null;
                      c = null;
                   }
-               } catch (IllegalAccessError | IOException | NoClassDefFoundError e) {
-                  // cannot get access to the class associated with the ClassLoader Asset
+               } catch (IOException | IndexOutOfBoundsException | LinkageError e) {
+                  // cannot get access to the class associated with the ClassLoaderAsset or
+                  // ByteArrayAsset
                   // do nothing
+                  log.warning("Failed to resolve servlet names for:" + key.get() + ". " + e.getMessage());
```

Comment:
`e.toString()` will probably be more useful here than `e.getMessage()` as it will include the name of the exception class.